NuGet.Config

```diff
@@ -3,7 +3,6 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
```

Comment:
I was definitely trying to remove this at one point but we still need it for the analyzer test packages. I requested they publish them to nuget.org but didn't get a commitment for them doing that.